lib/spack/spack/stage.py

```diff
@@ -932,7 +932,7 @@ def interactive_version_filter(
         colify(list(map(colorize, commands)), indent=2)
 
         try:
-            command = input(colorize("@*g{command>} ")).strip().lower()
+            command = input(colorize("@*g{top # or command>} ")).strip().lower()
```

Comment:
Can you make the message more clear?  I don't think people are going to understand "top #" in this context unless they recently used the original checksum interface -- and that's going to be fewer people over time.

Reply:
do you have a suggestion? i could expand the line `commands:` -> `enter a command or number of version to keep` or something, but then `command> ` looks weird as a prompt.